.github/workflows/build-pages-doxygen.yml

```diff
@@ -0,0 +1,62 @@
+# Based on NextJS sample build workflow
+#  https://github.com/actions/starter-workflows/blob/f3c5d7931d054ffbbdcbfdc463cc3bd0def74929/pages/nextjs.yml
+
+name: Deploy Pages and Doxygen
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["community"]
+    paths:
```

Comment:
do we want to automatically update the docs? if so this needs a new entry for `src/**`